ios/FluentUI/Other Cells/ActionsCell.swift

```diff
@@ -274,4 +282,12 @@ open class ActionsCell: UITableViewCell, TokenizedControlInternal {
         }
 
     }
+
+    private func setupBackgroundColors() {
+        if backgroundStyleType != .custom {
+            var customBackgroundConfig = UIBackgroundConfiguration.clear()
+            customBackgroundConfig.backgroundColor = backgroundStyleType.defaultColor(tokens: tokens)
```

Comment:
I'm not sure I understand why you're setting the customBackgroundConfig, instead of just clearing it and using the default backgroundConfiguration. This seems like an unexpected side effect behavior, but I might just not understand the full picture.

Reply:
i understand your confusion. one is customize from the client point of view. Fluent itself is also deviating from system tableviewcell configuration. so it has its own customconfig override its color based on fluent values.

Reply:
Ahh, I see. Maybe overrideBackgroundConfig?